deepchem/models/torch_models/torch_model.py

```diff
@@ -194,8 +195,8 @@ def __init__(self,
         if device is None:
             if torch.cuda.is_available():
                 device = torch.device('cuda')
-            elif torch.backends.mps.is_available():
-                device = torch.device('mps')
+            # elif torch.backends.mps.is_available():
```

Comment:
Can you remove this before merging in? Looks like cruft

Reply:
sure, forgot to remove. removed it.